packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -468,11 +469,22 @@ class AttachDisk extends Component {
   }
 }
 
-@connectStore(() => ({
-  checkPermissions: getCheckPermissions,
-  isAdmin,
-  allVbds: createGetObjectsOfType('VBD'),
-}))
+@addSubscriptions({
+  // used by getResolvedResourceSets
+  resourceSets: subscribeResourceSets,
```

Comment:
You can filter `resourceSets` here, so, [`getResolvedResourceSets`](https://github.com/vatesfr/xen-orchestra/blob/bb4e3dd96b9bc4c35dac26de7380b30d60ac7cda/packages/xo-web/src/xo-app/vm/tab-disks.js#L481) will not need to resolve unused resource sets and this [method](https://github.com/vatesfr/xen-orchestra/blob/bb4e3dd96b9bc4c35dac26de7380b30d60ac7cda/packages/xo-web/src/xo-app/vm/tab-disks.js#L497) will not be needed.